components/button/stories/button.stories.js

```diff
@@ -79,6 +80,16 @@ export default {
 			options: ["white", "black"],
 			control: "select",
 		},
+		showIconOnlyButton: {
+			table: {
+				disable: true,
+			},
+		},
+		showOneButtonPerLine: {
+			table: {
+				disable: true,
+			},
+		},
```

Comment:
```suggestion
		layout: {
		    name: "Layout",
		    description: "How the buttons align in the preview",
		    type: { name: "string" },
			table: { 
			    type: { summary: "string" },
			    category: "Advanced"
			},
			options: ["stacked","inline"],
			control: "radio"
		},
```

This might be nice to open up to developers for live-testing